website/public/locales/id/side_menu.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
We had some refactoring in another PR.  Sorry for another ask but can you move all these entries in to `common.json`?

Reply:
Thank you for letting me know about the refactor. I moved these into common file in [65e210a](https://github.com/LAION-AI/Open-Assistant/pull/1337/commits/65e210a0e2a93c15e854704971fa8204e4e6e94f) 👍